app/components/global/layout/SideNavLayout.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import { Flex, Box } from 'grid-styled'
+import { th } from '@pubsweet/ui-toolkit'
+
+const SideNavLink = styled(Link)`
```

Comment:
I'd recommend using [NavLink](https://reacttraining.com/react-router/web/api/NavLink) since this automatically sets the `active` class based on the URL.